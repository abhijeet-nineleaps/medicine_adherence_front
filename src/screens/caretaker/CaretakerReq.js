import {View} from 'react-native';
import React from 'react';

const CaretakerReq = () => {
  return <View />;
};

export default CaretakerReq;
