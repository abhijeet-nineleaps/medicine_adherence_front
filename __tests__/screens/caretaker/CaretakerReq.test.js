import React from 'react';
import renderer from 'react-test-renderer';
import CaretakerReq from '../../../src/screens/caretaker/CaretakerReq';

describe('Caretaker Request', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<CaretakerReq />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
