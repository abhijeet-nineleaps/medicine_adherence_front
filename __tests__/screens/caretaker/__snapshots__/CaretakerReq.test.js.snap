// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Caretaker Request renders correctly 1`] = `<View />`;
